src/Data/Audience.elm

```diff
@@ -34,7 +36,30 @@ type alias Audience =
     , folder : Maybe Int
     }
 
+{-| Decoder for an Audience
+-}
+decodeAudience : JD.Decoder Audience
+decodeAudience = JD.map4
+    Audience
```

Comment:
Just a nitpick remark - I was quite confused by this the first time I saw it - I assumed it's `elm-format` processed and so it didn't make sense to me how can `Audience ...` be a decoder -- but then I saw the `JD.map4` on the previous line :smile:

(We use elm-format everywhere in our codebase - and I can feel the benefits of having it right now on this anecdote :) less surprises when reading "foreign" code...)

Reply:
I am sorry, I should have definitely used `elm-format`.

Reply:
Never mind. It is always good to be aware of the things you are addicted on using :D E.g. how much you miss them when they are not used ;)